### ISSUE TITLE
--predict_ortho acting up and writing orthologs to file not working + No GO ids appear in .annotations file

### ISSUE THREAD

**Issue by selveyad** [eggnogdb/eggnog-mapper#213]
Howdy, 

I am receiving the error below:
```
Traceback (most recent call last):
  File "/home/aselvey/miniconda3/envs/eggnog/bin/emapper.py", line 1213, in <module>
    main(args)
  File "/home/aselvey/miniconda3/envs/eggnog/bin/emapper.py", line 279, in main
    dump_orthologs(seed_orthologs_file, orthologs_file, args)
  File "/home/aselvey/miniconda3/envs/eggnog/bin/emapper.py", line 812, in dump_orthologs
    for result in pool.imap(find_orthologs_per_hit, iter_hit_lines(seed_orthologs_file, args)):
  File "/home/aselvey/miniconda3/envs/eggnog/lib/python2.7/multiprocessing/pool.py", line 673, in next
    raise value
TypeError: 'NoneType' object has no attribute '__getitem__'
```
It occurs right after the functional annotation of refined hits finishes.

Here is my original command:
```
emapper.py -i ../trinity_assembly/Trinity.fasta.transdecoder_predict_all/Trinity.fasta.transdecoder.predict.all.pep --output T_td_pa_eggNOG -m diamond --cpu 16 --keep_mapping_files --target_taxa 'Lepidoptera' --target_orthologs all --predict_ortho --output_dir eggnog/
```
I was having trouble with the `--report_orthologs` option earlier. I removed that and got this error. I will remove `--predict_ortho` next and see what happens.

Also, if I want to resume a run, can I just put in the DIAMOND search results as my input file? I feel like I have been re-running the program for no reason haha. 

For anyone out there that had troubles with ete3 and the `--target_taxa` argument, you have to put the target taxa in **'** **'** (single quotes, no spaces) (ex. 'Lepidoptera') for it to work.

All help is greatly appreciated!

Best Regards,

Alex

** Update: emapper runs without error if `--report_orthologs` and `--predict_ortho` are not utilized. Whenever the patch for these options comes out I would really like to utilize them.

**Comment by selveyad** [comment]
New issue!

There are no GO ids in the GO column of the .annotations output. Same code as above was used. I am running the exact same command into a fresh outdir, will update once it is finished.

**Comment by Cantalapiedra** [comment]
Hi,

it could happen that you are not getting orthologs with GO annotation for the target taxa you are using?

**Comment by Cantalapiedra** [comment]
> 
> ** Update: emapper runs without error if `--report_orthologs` and `--predict_ortho` are not utilized. Whenever the patch for these options comes out I would really like to utilize them.

Those options should work in the "refactor" branch
